core/drivers/crypto/crypto_api/acipher/ecc.c

```diff
@@ -25,6 +25,7 @@ static size_t get_ecc_key_size_bytes(uint32_t curve)
 		return 28;
 
 	case TEE_ECC_CURVE_NIST_P256:
+	case TEE_ECC_CURVE_SM2:
```

Comment:
adding SM2 should be in a specific commit IMHO.

Reply:
I second that